src/api/firebase.js

```diff
@@ -166,7 +167,9 @@ export async function updateItem(listId, item) {
 	}
 }
 
-export async function deleteItem() {
+export async function deleteItem(listId, id) {
```

Comment:
```suggestion
export async function deleteItem(listId, itemId) {
```

Can we explicitly name the `id` parameter here? That way we know explicitly that's the input we want.